### DISCUSSION TITLE
Publish your extension to Microsoft Edge Addons website

### DISCUSSION THREAD

**Discussion by nagachaitanyalokam** [apollographql/apollo-client-devtools#580] [Ideas]
Microsoft Edge is the second most used browser on the web, making the browsing experience more flexible, offering many more services for users and optimized for Windows devices.

Microsoft Edge supports Chromium extensions, and you can publish your extensions to Microsoft Edge Add-ons website with minimal code changes. Find out more [here](https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/port-chrome-extension).

If you have any more queries about porting Chrome extension to Microsoft Edge or anything else, you can contact the relevant teams whose links are given [here](https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team). 

Let me know if this was helpful. I am happy to help you with more details.

**Comment by hwillson** [comment]
Thanks @nagachaitanyalokam - this is helpful. We're definitely considering this.

**Reply by lifeiscontent** [reply]
@hwillson please support edge!
